package com.hpbr.bosszhipin.network;

import b8.a;
import com.twl.http.config.RequestMethod;
import net.bosszhipin.base.BaseApiRequest;

/* JADX INFO: loaded from: classes7.dex */
public class GeekFilterListRequest extends BaseApiRequest<GeekFilterListResponse> {

    @a
    public String filterType;

    @a
    public String label;

    public GeekFilterListRequest(com.twl.http.callback.a<GeekFilterListResponse> aVar) {
        super(aVar);
    }

    @Override // com.twl.http.client.a
    public RequestMethod getMethod() {
        return RequestMethod.GET;
    }

    @Override // com.twl.http.client.a
    public String getUrl() {
        return com.hpbr.bosszhipin.a.I5;
    }
}